docs_src/src/pages/documentation/api_reference/cors.mdx

```diff
@@ -21,14 +21,14 @@ You can allow CORS for your application by adding the following code:
       from robyn import Robyn, ALLOW_CORS
 
       app = Robyn(__file__)
-      ALLOW_CORS(app)
+      ALLOW_CORS(app, origins = "http://localhost:<PORT>/")
```

Comment:
Hey @carlosm27 👋 

Thank you for your PRs ✨ but origins is a `list[str]` and not just a `str`